### ISSUE TITLE
New Pokémon TV Player Support

### ISSUE THREAD

**Issue by klop422** [ytdl-org/youtube-dl#22517]
## Checklist

- [x] I'm reporting a new site support request
- [x] I've verified that I'm running youtube-dl version **2019.09.12.1**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [x] I've searched the bugtracker for similar site support requests including closed ones

## Example URLs

- Single video: https://watch.pokemon.com/en-gb/player.html?id=1b0e462fc0184fbfb8d239956c0e0e4f
- Single video: https://watch.pokemon.com/de-de/player.html?id=b85ebd49197e49259c4f01780b0585f7

## Description

Pokémon TV got updated and is no longer supported by youtube-dl. Would it be possible to add support for the new version of the site?

**Comment by KevTheRev13** [comment]
I also am here for this same issue. Any updates to this?

**Comment by blackjack4494** [comment]
@KevTheRev13 added to my fork. you can use it there :)

**Comment by KevTheRev13** [comment]
Hi @blackjack4494 . Thanks.
I actually ended up running the code from someone else's fork back in July, and have had the functionality that way.
See here for more details https://github.com/ytdl-org/youtube-dl/pull/22626#issuecomment-686218287

It was @Masterditto that may be still be interested in a more user friendly way to get the functionality. Thank you though.

**Comment by blackjack4494** [comment]
@KevTheRev13  Yes I also replied to him. Update will come out within next 1-2 hours. But quite nice of you that you kept @Masterditto in mind.

**Comment by tqoo** [comment]
Hi all, and sorry for replying to a mostly dead thread.
Despite using both @blackjack4494's solution and @ssaqua's solution, I keep getting the Unsupported URL Error. Please help!
